src/components/pages/Tests/TestsList/TestCreationModalContent/TestCreationForm.tsx

```diff
@@ -113,7 +113,7 @@ const TestCreationForm: React.FC<TestCreationFormProps> = props => {
           rules={[required, k8sResourceNamePattern, k8sResourceNameMaxLength]}
           required
         >
-          <Input placeholder="e.g.: my-test" />
+          <Input placeholder="e.g.: my-test" autoComplete="off"/>
```

Comment:
Did you test it in multiple browsers? As I wrote in https://github.com/kubeshop/testkube/issues/400, `autoComplete` with an unknown value (i.e. `nope`) works more often. The implementation is changing over time though, so I'm not 100% sure if it's best at this very moment.

```suggestion
          <Input placeholder="e.g.: my-test" autoComplete="nope" aria-autocomplete="none" />
```

Reply:
It worked like 7-8 times in Edge and chrome, I test it with autocomplete to nope